packages/block-library/src/latest-posts/constants.js

```diff
@@ -1,3 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	alignNone,
+	positionCenter,
+	positionLeft,
+	positionRight,
+} from '@wordpress/icons';
+
 export const MIN_EXCERPT_LENGTH = 10;
 export const MAX_EXCERPT_LENGTH = 100;
 export const MAX_POSTS_COLUMNS = 6;
+
+export const BLOCK_ALIGNMENTS_CONTROLS = [
+	{ value: undefined, label: __( 'None' ), icon: alignNone },
+	{ value: 'left', label: __( 'Align left' ), icon: positionLeft },
+	{
+		value: 'center',
+		label: __( 'Align center' ),
+		icon: positionCenter,
+	},
+	{ value: 'right', label: __( 'Align right' ), icon: positionRight },
+];
```

Comment:
Not sure if we should expose a native-only constant in the constants file, if it's going to be used only in the `edit.native.js` file I might consider defining the constant there, wdyt?

Reply:
The changes to this shared constants file have been removed following the recent refactor. 👍